app/controllers/api/conversion_hosts_controller.rb

```diff
@@ -1,5 +1,103 @@
 module Api
   class ConversionHostsController < BaseController
     include Subcollections::Tags
+
+    # Whitelist of valid resource types
+    VALID_TYPES = {
```

Comment:
Wow, this is much more restrictive than I thought.  I'm glad we added this.

Reply:
Yep, I read an article or two that convinced me. Thanks for bringing it up.